content/javascript/concepts/dates/dates.md

```diff
@@ -1,141 +1,102 @@
 Subjects:
   - 'Web Development'
   - 'Computer Science'
 Tags:
-  - 'Date'
   - 'Data Types'
+  - 'Date'
+  - 'Time'
 CatalogContent:
   - 'introduction-to-javascript'
   - 'paths/create-a-back-end-app-with-javascript'
 ---
 
-`Date` objects in JavaScript represent a single moment in time stored as a number that represents the number of milliseconds since midnight on January 1st 1970 UTC. Note that this is different from the UNIX epoch, the number of _seconds_ since midnight on January 1st 1970 UTC.
+In JavaScript, `Date` objects represent a single moment in time stored as a number, representing the number of milliseconds since midnight on January 1st 1970 UTC.
 
-Also note that, even though the representation of a `Date` is in UTC, the methods to fetch a date or its components work in the host system's local time zone.
-
-The defined range for a `Date` value is between April 20, 271821 BCE to September 13, 275760 CE.
+**Note:** This is different from the UNIX epoch, the number of _seconds_ since midnight on January 1st 1970 UTC. Even though the representation of a `Date` is in UTC, the methods to fetch a date or its components work in the host system's local time zone (which may be different from UTC).
 
 ## Syntax
 
-When used as a function, `Date()` returns the current date and time. When used as a constructor, `Date()` returns a new date object.
+```pseudo
+const myDate = new Date(dateValue);
+```
 
-```javascript
-var now = Date();
+The `dateValue` can be either of the following:
 
-console.log(now);
-// Output: Current date as a string
-// Wed July 28 2021 16:47:34 GMT+0000 (Coordinated Universal Time)
+- It can be a [string](https://www.codecademy.com/resources/docs/javascript/strings) representation of a date that is valid and [IETF-compliant](https://datatracker.ietf.org/doc/html/rfc2822#page-14)
+- It can be another instance of the `Date` class.
+- It can be an integer that represents the date measured in milliseconds since January 1st 1970 UTC.
+
+The defined range for a `Date` value is between April 20, 271821 BCE to September 13, 275760 CE. Individual datetime elements can be defined in the `Date()` constructor with the following syntax:
+
+```pseudo
+const d = new Date(year, month, day, hour, minute, second, millisecond);
+```
+
+### Date Format Types
+
+In JavaScript, the following three date format types are commonly used:
+
+|              Format Type              | Syntax                                                  | Example Output                                              |
+| :-----------------------------------: | ------------------------------------------------------- | ----------------------------------------------------------- |
+| ISO Date (The International Standard) | `date.toISOString()`                                    | "2015-04-05T00:00:00.000Z"                                  |
```

Comment:
The syntax/example for the ISO date are inconsistent with the other format types. I'd expect something like "YYYY-MM-DDTHH:mm:SS.SSSZ"

Also, example output is not as useful as example inputs. i.e. Examples of actual date strings you'd plug into date()